packages/medusa/src/api/admin/orders/[id]/fulfillments/[fulfillment_id]/shipment/route.ts

```diff
@@ -0,0 +1,40 @@
+import { createOrderShipmentWorkflow } from "@medusajs/core-flows"
+import { remoteQueryObjectFromString } from "@medusajs/utils"
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "../../../../../../../types/routing"
+import { AdminOrderCreateShipmentType } from "../../../../validators"
+
+export const POST = async (
+  req: AuthenticatedMedusaRequest,
+  res: MedusaResponse
+) => {
+  const remoteQuery = req.scope.resolve("remoteQuery")
+
+  const variables = { id: req.params.id }
+
+  const input = {
+    ...(req.validatedBody as AdminOrderCreateShipmentType),
+    order_id: req.params.id,
+    fulfillment_id: req.params.fulfillment_id,
+  }
+
+  const { errors } = await createOrderShipmentWorkflow(req.scope).run({
+    input,
+    throwOnError: false,
+  })
+
+  if (Array.isArray(errors) && errors[0]) {
+    throw errors[0].error
+  }
```

Comment:
wdyt if by default the workflows always re throw the first error? And if you want to handle it manually then you can set the flag and check how many errors they are and how to handle them?